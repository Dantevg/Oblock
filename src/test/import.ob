{
	Vector = import "Vector"
	a = Vector.clone { x = 10; y = 20 }
	b = Vector.clone { x = 5; y = 2 }
	c = a + b
	d = a * b
	print(c.toString(), d)
}
