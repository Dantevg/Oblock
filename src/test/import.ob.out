Vector (15, 22)	90
